js/main.js

```diff
@@ -0,0 +1,23 @@
+const getRandomNumber = (a, b) => {
+  if (a<0 || b<0) {
+    console.log('ошибка, число меньше 0');
+    return 0;
+  }
+  if (b<=a) {
+    console.log('ошибка, число "до" меньше или равно числу "от"');
+  }
+  return Math.round(Math.random() * (b - a) + a);
+}
+
+const getRandomDeciamlNumber = (a, b, c=1) => {
```

Comment:
Опечатка в decimal. И обычно для дробей используется float в названии. Все остальные замечания для 1 функции справедливы так же и для этой.
```suggestion
const getRandomFloat = (a, b, c = 1) => {
```